cmd/ak/cmd/sessions/log.go

```diff
@@ -50,6 +53,7 @@ func init() {
 	// Command-specific flags.
 	logCmd.Flags().IntVarP(&skip, "skip", "s", 0, "number of entries to skip")
 	logCmd.Flags().BoolVarP(&noTimestamps, "no-timestamps", "n", false, "omit timestamps from track output")
+	logCmd.Flags().BoolVarP(&justPrints, "just-prints", "p", false, "print only log entries with print messages")
```

Comment:
"prints-only" sounds better than "just-prints", and a more accurate description

```suggestion
	logCmd.Flags().BoolVarP(&justPrints, "prints-only", "p", false, "output only workflow print messages")
```

Reply:
fixed